application/features/analytics.feature

```diff
@@ -3,9 +3,11 @@ Feature: User
   In order to report about mfa usage
   I need to be able to count active users with diffent kinds of verified mfa records
 
+  Background:
+    Given the user store is empty
+
   Scenario: Get a count of active users with a verified mfa
-    Given that no mfas or users exist
-      And I create a new user
+      Given I create a new user
```

Comment:
As another side note about Behat/Gherkin, if you can find ways to make the `Given`'s declarative rather than imperative statements (e.g. "1 user exists who has ____" rather than "I create a new user"), it seems to result in more readable tests. The `Given`'s are simply setting up the state of the system before it's tested, whereas the `When` is where we want a specific action to be taken.

That said, I realize you're simply adding to and/or revising an existing set of tests, and it certainly makes sense to use existing steps rather than write new steps unnecessarily. Feel free to proceed along your existing route; I just wanted to share some things I've learned, for future code.

Reply:
Good thoughts. Thanks for sharing!